app/models/rent.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Rent < ApplicationRecord
+  validates :user, :book, :from, :to, presence: true
+  validates_date :to, after: :from, after_message: 'must be after the "from" date'
```

Comment:
lets validate the dates with a custom method, you can write a method that checks the dates and returns a boolean value, and include it in the validations.